From da5c9de6ed20219ae616080c9146735426f296d3 Mon Sep 17 00:00:00 2001
From: Takashi Sakamoto <o-takashi@sakamocchi.jp>
Date: Mon, 6 Feb 2023 09:59:02 +0900
Subject: [PATCH] hwdb: ieee1394-unit-function: add MOTU 896 mk3 Hybrid

This commit adds hwdb entry for MOTU 896 mk3 Hybrid.

Reference: https://lore.kernel.org/alsa-devel/63DF98FE.2060604@gmx.de/
(cherry picked from commit e478f8c04cd70821be470e848bfc27959c320b37)

Related: RHEL-5950
---
 hwdb.d/80-ieee1394-unit-function.hwdb | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hwdb.d/80-ieee1394-unit-function.hwdb b/hwdb.d/80-ieee1394-unit-function.hwdb
index 6fa1329171..cf55f5524c 100644
--- a/hwdb.d/80-ieee1394-unit-function.hwdb
+++ b/hwdb.d/80-ieee1394-unit-function.hwdb
@@ -1094,6 +1094,13 @@ ieee1394:ven000001F2mo*sp000001F2ver00000035
  IEEE1394_UNIT_FUNCTION_MIDI=1
  IEEE1394_UNIT_FUNCTION_AUDIO=1
 
+ieee1394:node:ven0x0001f2units0x0001f2:0x000037
+ieee1394:ven000001F2mo00102800sp000001F2ver00000037
+ ID_VENDOR_FROM_DATABASE=MOTU
+ ID_MODEL_FROM_DATABASE=MOTU 896 mk3 Hybrid
+ IEEE1394_UNIT_FUNCTION_MIDI=1
+ IEEE1394_UNIT_FUNCTION_AUDIO=1
+
 ieee1394:node:ven0x0001f2units0x0001f2:0x000039
 ieee1394:ven000001F2mo*sp000001F2ver00000039
  ID_VENDOR_FROM_DATABASE=MOTU
